### PR TITLE
added reorder to profiler

### PR DESCRIPTION
**Pull request by jakpiase**
### PR types
New features 

### PR changes
Others

### Describe
Added reorder to profiler

### CODE REVIEW

**Comment by jakpiase** [comment]
Add @luotao1 could you please advise why this failure happens?
PR-CI-APPROVAL

**Review by jczaja — DISMISSED** [review]
LGTM

**Comment by jczaja** [comment]
@luotao1 Could you please advice on failing PR-CI-APPROVAL and start your review ?   

**Note:** Changes are in profiler of PaddlePaddle

**Comment by luotao1** [comment]
Please pass all the CIs at first except APPROVAL

File: paddle/fluid/framework/data_layout_transform.cc [hunk 1]
```diff
@@ -194,6 +194,9 @@ void innerTransDataLayoutFromMKLDNN(DataLayout in_layout, DataLayout out_layout,
         handler.AcquireReorder(reorder_dst_memory_p, reorder_src_memory_p);
 
     mkldnn::stream astream(cpu_engine);
+    #ifdef PADDLE_WITH_MKLDNN
```
Comment (wojtuss) [inline paddle/fluid/framework/data_layout_transform.cc:197]: I think we do not need this guard. We are already in a section executed with MKLDNN.

**Comment by jakpiase** [comment]
I have created other PR with merged internal and external additions.

**Pull request closed without merging**